src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -1038,6 +1038,33 @@ public Metadata createMetadata(String typeName, String scope, Metadata metadata)
         return new Metadata(JsonObject.readFrom(response.getJSON()));
     }
 
+    /**
+     * Attempts to create a metadata value on a file. If conflict then perform an update of metadata values on file.
```

Comment:
This description could probably be cleaned up to be more relevant for developers; it describes at a technical level what the method does, but I'm not sure that's what users are likely to care about.  Perhaps something like "Sets the provided metadata on the file, overwriting any existing metadata keys already present" could convey what someone should expect to happen as the end result of calling the method?